### ISSUE TITLE
Would you accept entries for Afrikaans sites in your list?

### ISSUE THREAD

**Issue by DandelionSprout** [EasyDutch-uBO/EasyDutch#41]
### Prerequisites / Vereisten

- [X] I performed a cursory search of the issue tracker to avoid opening a duplicate issue / Geen **duplicaten**!!

### Your Question(s) / Uw vraag

There are currently no lists out there that cover Afrikaans sites, with the situation being as following:
• EasyList only covers English-language sites.
• uBlock Filters appears to only cover English-language sites.
• AdGuard Base/Dutch thinks none of the 4 Afrikaans sites I found that had ads on them, had [high enough Alexa ranks to be worth fixing](https://github.com/AdguardTeam/AdguardFilters/issues/149597).
• I remember from maybe 2020 that EasyList Dutch wasn't very keen on fixing Afrikaans sites either.
• EasyDutch has been [enabled by default](https://github.com/gorhill/uBlock/blob/master/assets/assets.json#L581) in uBO on web browsers whose language was set to Afrikaans, admittedly as the result of me expanding browser ISO tag coverage in uBO in 2021.

I did some Dutch/Afrikaans text comparison and saw that their word spellings had substantial differences, but hopefully not to the point of incomprehension for Dutch speakers.

If you were to say yes, I am aware of 4 sites that'd be affected by this, along with my entry proposals for them:
```
! https://diecourant.co.za/
diecourant.co.za##section[id^=smartslider]
diecourant.co.za###secondary
! https://diecourant.co.za/2023/01/23/sekondere-skoolonderrig-so-lyk-2023-se-fooie/
||diecourant.co.za/wp-content/uploads/*-20*-1024x*.jpg^$image
! https://overstrandherald.co.za/
overstrandherald.co.za##.wpb_widgetised_column .widget_text
! https://www.republikein.com.na/
republikein.com.na##.topBanner
republikein.com.na##.hidden-xs > aside:has(a[href*="//nmh.my.na/advert"])
republikein.com.na##aside[style] ~ aside:not([style])
! https://www.republikein.com.na/landbou/vere-waai-oor-hoenderprodukte2023-04-28
republikein.com.na##.articleBody > aside
! https://www.pretoriafm.co.za/nuus/
pretoriafm.co.za##.bannerswidget
```

**Comment by BPower0036** [comment]
Yes, as long as South African sites are not in English, but in Afrikaans, we will fixe them.

**Comment by DandelionSprout** [comment]
Good to hear. Should I file individual reports for the 4 Afrikaans-only sites above, or is the info above already sufficient?

**Comment by BPower0036** [comment]
The above info is sufficient.

**Comment by BPower0036** [comment]
Fixed by #42
